websites/Y/YomuMangás/metadata.json

```diff
@@ -0,0 +1,72 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "293913134748401674",
+		"name": "Vitu™#2002"
+	},
+	"service": "YomuMangás",
+	"description": {
+		"en": "Read portuguese mangas and doujinshis online for free without ads at YomuMangás",
+		"pt_BR": "Leia mangás e doujinshis, de graça e sem anúncios na YomuMangás."
+	},
+	"url": [
+		"yomumangas.com",
+		"dev.yomumangas.com"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/RBkqiAg.png",
+	"thumbnail": "https://i.imgur.com/EqYC4Qo.png",
+	"color": "#121212",
+	"category": "anime",
+	"tags": [
+		"anime",
+		"manga",
+		"doushinji",
+		"geek",
+		"nsfw"
+	],
+	"settings": [
```

Comment:
Let's put privacy at the top and if you want it to stop showing images etc when privacy is on check these settings

https://github.com/PreMiD/Presences/blob/main/websites/Y/YouTube/metadata.json#L96-L105